### PR TITLE
changed inherited Axes calls to super

### PR DESCRIPTION
**Pull request by ordirules**
Changed Axes bound method calls using super.
Just one command: 
vim regexp: 

%s;Axes\\.\\(.*\\)(self, *;super(Axes3D, self).\\1(;g
followed by:
%s;Axes.\\(.*\\)(self *;super(Axes3D, self).\\1(;g

This came form #8455 . Did not have time to test, waiting for Travis.

## PR Checklist
- [ ] Has Pytest style unit tests
- [x] Code is PEP 8 compliant 
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/whats_new.rst if major new feature
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by ordirules** [comment]
oops Regexp was incorrect, modified and force pushed new branch with corrected version.

File: lib/mpl_toolkits/mplot3d/axes3d.py [hunk 1]
```diff
@@ -95,11 +95,11 @@ def __init__(self, fig, rect=None, *args, **kwargs):
             self._shared_z_axes.join(self, sharez)
             self._adjustable = 'datalim'
 
-        Axes.__init__(self, fig, rect,
+        super(Axes3D, self).__init__(fig, rect,
```
Comment (anntzer) [inline lib/mpl_toolkits/mplot3d/axes3d.py]: fix indent

Comment (ordirules) [inline lib/mpl_toolkits/mplot3d/axes3d.py]: thanks!

**Comment by ordirules** [comment]
Any idea why Travis is failing in python 2.7?
```
>                % err)

E           ImageComparisonFailure: images not close (RMS 0.104):

E           	result_images/test_mplot3d/mixedsubplot_svg.png

E           	result_images/test_mplot3d/mixedsubplot-expected_svg.png
```

**Comment by ordirules** [comment]
@OceanWolf I forgot to ping you on this

**Comment by tacaswell** [comment]
The mixed subplot test is know to be flaky, but we have not tracked down why yet.

I suspect this conflicts with your other PR that just got merged.

**Comment by ordirules** [comment]
yeah, that's right. rebased (and changed the long title...)
Hopefully Travis won't complain.
thanks!

File: lib/mpl_toolkits/mplot3d/axes3d.py [hunk 2]
```diff
@@ -208,11 +209,12 @@ def _init_axis(self):
             ax.init3d()
 
     def get_children(self):
-        return [self.zaxis, ] + Axes.get_children(self)
+        return [self.zaxis, ] + super(Axes3D, self).get_children()
 
     def _get_axis_list(self):
         return super(Axes3D, self)._get_axis_list() + (self.zaxis, )
 
+
```
Comment (QuLogic) [inline lib/mpl_toolkits/mplot3d/axes3d.py]: Extra line.

**Comment by ordirules** [comment]
squashed. but author is a different email. i'll have to do that a little later.

**Comment by ordirules** [comment]
fixed. On one machine I was working on, I had an old email as author. Thanks for catching this subtle but important detail @QuLogic

**Comment by ordirules** [comment]
Travis failed on py3.6. Do you know if this is a flakey test?

**Comment by jenshnielsen** [comment]
Yes that test measures performance and can fail do to travis workload

**Pull request merged**